bot/modules/stats.js

```diff
@@ -33,18 +33,24 @@ exports.stats = {
           day_indicator = ':thumbsdown:';
         }
 
-        needle.get('https://api.coinmarketcap.com/v2/ticker/1298/?convert=GBP', function(error, response) {
+        needle.get('https://api.coingecko.com/api/v3/coins/markets?vs_currency=gbp&ids=lbry-credits&order=market_cap_desc&per_page=100&page=1&sparkline=false', function(error, response) {
           if (error || response.statusCode !== 200) {
             msg.channel.send('coinmarketcap API is not available');
           } else {
             data = response.body.data;
-            let price_gbp = Number(data.quotes.GBP.price);
-            needle.get('https://api.coinmarketcap.com/v2/ticker/1298/?convert=EUR', function(error, response) {
+            let price_gbp = Number(data.current_price);
+            needle.get('https://api.coingecko.com/api/v3/coins/markets?vs_currency=eur&ids=lbry-credits&order=market_cap_desc&per_page=100&page=1&sparkline=false', function(error, response) {
               if (error || response.statusCode !== 200) {
                 msg.channel.send('coinmarketcap API is not available');
               } else {
                 data = response.body.data;
-                let price_eur = Number(data.quotes.EUR.price);
+                let price_eur = Number(data.current_price);
+                needle.get('https://api.coingecko.com/api/v3/coins/markets?vs_currency=usd&ids=lbry-credits&order=market_cap_desc&per_page=100&page=1&sparkline=false', function(error, response) {
+              if (error || response.statusCode !== 200) {
+                msg.channel.send('coinmarketcap API is not available');
```

Comment:
this should say coingecko, same for a few others above

Reply:
also the variable name is `price_eur` but it's usd

Reply:
@nikooo777 resolved on https://github.com/lbryio/lbry-wunderbot/pull/266/commits/5f825b9156594b49f1eefdf57e29138d898c2b2c